clang/test/CIR/CodeGen/multi-vtable.cpp

```diff
@@ -86,7 +86,7 @@ int main() {
 // LLVM-DAG: @_ZTVN10__cxxabiv117__class_type_infoE = external global ptr
 
 //   typeinfo name for Mother
-// CIR: cir.global linkonce_odr @_ZTS6Mother = #cir.const_array<"6Mother" : !cir.array<!s8i x 7>> : !cir.array<!s8i x 7> {alignment = 1 : i64}
+// CIR: cir.global linkonce_odr @_ZTS6Mother = #cir.const_array<"6Mother" : !cir.array<!s8i x 7>> : !cir.array<!s8i x 7> {alignment = 1 : i64, comdat}
```

Comment:
Should be `cir.global linkonce_odr comdat @_ZTS6Mother ...` instead. No default printing at the end.